finality-aleph/src/aggregation/mod.rs

```diff
@@ -1,14 +1,40 @@
-use aleph_bft::SignatureSet;
-use aleph_bft_rmc::Message;
+use std::{fmt::Debug, hash::Hash, marker::PhantomData, time::Instant};
+
+use aleph_aggregator::NetworkError;
+use aleph_bft::{Recipient, SignatureSet};
 use sp_runtime::traits::Block;
 
-use crate::crypto::Signature;
+use crate::{
+    crypto::Signature,
+    metrics::Checkpoint,
+    network::{Data, DataNetwork, SendError},
+    Metrics,
+};
 
-mod aggregator;
-mod multicast;
+pub type RmcNetworkData<B> =
+    aleph_aggregator::RmcNetworkData<<B as Block>::Hash, Signature, SignatureSet<Signature>>;
 
-pub use aggregator::{BlockSignatureAggregator, IO};
-pub use multicast::SignableHash;
+pub struct NetworkWrapper<D: Data, N: DataNetwork<D>>(pub N, pub PhantomData<D>);
```

Comment:
Why is the `PhantomData` `pub`? :D

Reply:
because I was lazy and didnt want to implement constructor for this :P fixed